prow/scripts/README.md

```diff
@@ -0,0 +1,21 @@
+# Cluster
+
+## Overview
+
+The folder contains various scripts involved in integration tests.
+
+### Project structure
+
+<!-- Update the folder structure each time you modify it. -->
+
+The structure of the folder looks as follows:
+
+```
+  ├── cluster-integration               # Scripts for the cleanup of Google Cloud Platform (GCP) objects after completion of integration tests.             
+  ├── create-dns-entry.sh               # The script that creates a DNS entry on a GKE cluster for integration tests.
+  ├── generate-self-signed-cert.sh      # The script for creating a self-signed certificate on a GKE cluster for integration tests.
```

Comment:
It only creates certificates locally. Read description of script: https://github.com/kyma-project/test-infra/blob/master/prow/scripts/generate-self-signed-cert.sh